src/server.py

```diff
@@ -50,6 +50,9 @@ def game_starter_thread(self):
         # Broadcast to all players that the game has started TODO: how do we make sure all player threads have unblocked at this point?
         self.broadcast("game starts", s.encode_startgame())
 
+        for address in self.__state.get_all_socket_addr():
+            self.__state.player_signal_start_game(address)
```

Comment:
the only concern I have for now (but i don't have a solution yet) is how is this delay (between clients already start on the game while player_listener threads on the server side still block for) gonna have an effect on the game state.